peps/pep-0756.rst

```diff
@@ -3,11 +3,12 @@ Title: Add PyUnicode_Export() and PyUnicode_Import() C functions
 Author: Victor Stinner <vstinner@python.org>
 PEP-Delegate: C API Working Group
 Discussions-To: https://discuss.python.org/t/63891
-Status: Draft
+Status: Rejected
 Type: Standards Track
 Created: 13-Sep-2024
 Python-Version: 3.14
 Post-History: `14-Sep-2024 <https://discuss.python.org/t/63891>`__
+Resolution: `29-Oct-2024 <https://github.com/capi-workgroup/decisions/issues/44#issuecomment-2444841064>`__
```

Comment:
We don't support GitHub links in the resolution header -- this is a suggestion to fix, but feel free to re-word.

```suggestion
Resolution: 29-Oct-2024

.. note:: Withdrawl

   This PEP has been withdrawn__ by the author.
   
   __ https://github.com/capi-workgroup/decisions/issues/44#issuecomment-2444841064
```